### ISSUE TITLE
Hyperparameter suggestions for large scenes?

### ISSUE THREAD

**Issue by hvkwak** [NVlabs/neuralangelo#117]
Hi, thank you very much for your incredible work. I am trying to reconstruct a large scene of my custom dataset, where I used the hyperparameters of `dict_size=22`, `dim=8` with all camera poses downward watching. Although this is a modular approach(which reconstructs only a part of the scene), the quality of the extracted mesh is not quite promising. Here's an example after the iteration of 300K: 

I have got two questions:
- May more iterations be helpful?
- Any hyperparameter suggestions for better reconstruction of large scenes?

**Comment by mli0603** [comment]
Hi @hvkwak 

What is your batch size for the data?

**Comment by hvkwak** [comment]
> Hi @hvkwak
> 
> What is your batch size for the data?

Hi, I kept it 1. Let me train this again with the batch_size of 16. Thank you!

**Comment by hvkwak** [comment]
> Hi @hvkwak
> 
> What is your batch size for the data?

Hi @mli0603, oops! I kept it 1. Let me train this again with the batch_size of 16. Thank you!

**Comment by hvkwak** [comment]
@iam-machine @mli0603  The batch_size of 16 was somewhat a better idea, but recovering correct geometry of the scene is not promising with the following parameters per iteration:
-rand_rays: 1024 (randomly sampled from all images, not just batch_size of 16.)
-coarse: 512
-fine: 128

The reconstructed scene is flat. My camera poses are all just downward watching from the sky, all of which is almost equally distant from the ground level. No camera poses are near the ground level, nor close-up shot images are available. I disabled background. Would you have any suggestions?
![IMG_1160](https://github.com/NVlabs/neuralangelo/assets/39279060/acbe49ed-c2ca-48ba-aa70-420972da95f9)

**Comment by hvkwak** [comment]
@iam-machine  @mli0603 Batch size was the key idea to reconstruct large scenes (more specifically 512 randomly sampled rays from all images per iteration), where coarse = 512, fine = 128 captures the geometry well. Thank you for your support.

**Comment by hvkwak** [comment]
@iam-machine @mli0603 Batch size was the key idea to reconstruct large scenes (more specifically 512 randomly sampled rays from all images per iteration), where coarse = 512, fine = 128 capture the geometry well. Thank you for your support.

**Comment by camolina2** [comment]
Hi @hvkwak, in which file did you change the params? can you write it here for me please. 

Thank you in advance!
